File: ocaml-3.10.2/test-test.ml

let 
 checkComplainAndAdjustExpected= Testenv.TestEnv.checkComplainAndAdjustExpected and
 test= Testenv.TestEnv.test and
 result= Testenv.TestEnv.result;;


checkComplainAndAdjustExpected 0;;
test true;;
test true;;
checkComplainAndAdjustExpected 2;;
test true;;
test true;;
test true;;
checkComplainAndAdjustExpected 3;;

Printf.printf "%s\n" (result ());;

File: python-2.6.4/TestTest.py
#!/usr/bin/python
# -*- coding: utf-8 -*-
# vi:encoding=utf-8

from TestEnv import TestEnv

te= TestEnv()

te.checkComplainAndAdjustExpected( 0)

te.test( True)
te.test( True)

te.checkComplainAndAdjustExpected( 2)

te.test( True)
te.test( True)
te.test( True)

te.checkComplainAndAdjustExpected( 3)

print te.result()
